reana_db/models.py

```diff
@@ -178,6 +178,7 @@ class User(Base, Timestamp, QuotaBase):
         sync_backref=False,
     )
     audit_logs = relationship("AuditLog", backref="user_")
+    idp_id = Column(String(length=255))
```

Comment:
I hope that the IdP ID limit of max 255 characters will suffice in general. Should be the case, but can you please quick-check the various popular AAI systems out there in order to be fully sure?